DidIFloss/Views/HomeScreenView.swift

```diff
@@ -11,45 +11,52 @@ struct HomeScreenView: View {
     
     var body: some View {
         
-        ZStack {
-            Color.flossSkyBlue
-            
-            homeScreenImageContent
-            
-            homeScreenTextContent
+        NavigationView {
+            ZStack {
+                
+                Color.flossSkyBlue
+                
+                toothExactlyPositions
+                
+                homeScreenTextContent
+                
+            }
+            .ignoresSafeArea()
         }
-        .ignoresSafeArea()
     }
     
     var homeScreenTextContent: some View {
         
         VStack {
             Spacer()
             
-            VStack(spacing: -60) {
+            VStack(spacing: -50) {
                 Text("did I")
                 Text("floss?")
             }
-            .font(.custom(Constants.FontNames.borel, size: 90))
-            .foregroundStyle(Color.flossLightYellow)
+            .font(.custom(Constants.FontNames.borel, size: 70))
```

Comment:
Nada errado, mas considere usar viewModifiers customizados ao utilizar muitos modificadores de uma só vez, principalmente se for algo "padrão" ou existem motivos para crer que esse estilo poderá ser utilizado em outros pontos da aplicação. Outro motivo interessante é para melhorar a legibilidade do código